src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Lookup_V1Compat.txt

```diff
@@ -0,0 +1,10 @@
+﻿#SETUP: DisableMemChecks,TableSyntaxDoesntWrapRecords,PowerFxV1CompatibilityRules
+
+// Untyped blanks are not allowed in the argument that defines the row scope
+>> LookUp(Blank(), Blank())
```

Comment:
>ookUp(Blank(), Blank() [](http://example.com/codeflow?start=4&length=22)

This is a breaking change: we won't allow untyped blanks to define the row scope. There isn't a real scenario where this would be useful, besides language semantic tests.